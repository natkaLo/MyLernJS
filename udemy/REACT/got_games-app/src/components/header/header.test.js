import React from "react";
import Header from './header';
import {shallow} from 'enzyme';
describe('Testing<Header/>', () => {
    //тест кейс. Начинается с ключевого слова it
    it('Header have rendered correctly', () => {
        //создаем компонент который надо протестировать. Создаем с помощью renderer из библиотеки react-test-renderer
        const header = shallow(<Header/>); //получаем джейсон объект
        //хотим создать снимок как этот компонент должен выглядеть
        expect(header).toMatchSnapshot();
        //запуск теста - в терминале
        //npm run test

    })
});