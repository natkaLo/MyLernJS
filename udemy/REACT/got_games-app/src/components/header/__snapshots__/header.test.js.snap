// Jest Snapshot v1, a

exports[`Testing<Header/> Header have rendered correctly 1`] = `ShallowWrapper {}`;
